Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -1201,3 +1214,13 @@ void ABenchmarkGymGameModeBase::OnMemReportFlagUpdate(const FString& FlagName, c
 	InitMemReportTimer(FlagValue);
 #endif
 }
+
+void ABenchmarkGymGameModeBase::GetMetrics(const FString& LeftLabel, const FString& RightLabel, const FString& MetricsName, ABenchmarkGymGameModeBase::FunctionPtrType Func)
+{
+	auto MetricsPtr = UMetricsBlueprintLibrary::GetMetric(MetricsName, TArray<FPrometheusLabel>{ TPair<FString, FString>(LeftLabel, RightLabel), TPair<FString,FString>(MetricEnginePlatformLeftLabel,MetricEnginePlatformRightLabel) });
+	if (MetricsPtr != nullptr)
```

Comment:
Omega Nit: Think we subtly prefer .IsValid() checks for smart pointers.

Reply:
make sense